export default toast to reuse them with custom props. (#225)
calintamas/react-native-toast-message: README.md

If you want to add custom types - or overwrite the existing ones - you can add a `config` prop when rendering the `Toast` in your app `root`. 

> You can either use the default `BaseToast` style and adjust its layout, or create Toast layouts from scratch or reuse the default layout

Reviewer: Can you adjust the wording please?

```suggestion
You can either use any of the default `BaseToast`, `SuccessToast`, `ErrorToast` or `InfoToast` components and adjust their layout, or create Toast layouts from scratch.
```

    />
  ),
  

>   /*
>     Reuse the default toast with custom props.
>   */
>   error: ({ hide, ...rest }) => (
>         <ErrorToast {...rest} onTrailingIconPress={hide} 
>             text1Style={{
>                 fontSize: 17
>             }}
>             text2Style={{
>                 fontSize: 15
>             }}
>         />
>     ),
>   

Reviewer: Here as well

```suggestion
   /*
    Reuse the default ErrorToast toast component
  */
  error: (props) => (
    <ErrorToast 
      {...props}
      text1Style={{
        fontSize: 17
      }}
      text2Style={{
        fontSize: 15
      }}
    />
  ),
```
